Asciidoclet failing to load AsciidoctorJ dependency

version: 2.0.0-snapshot version in main branch

The current version works when using the `shaded` jar, otherwise `mvn javadoc:javadoc` failw with

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.6.3:javadoc (default-cli) on project asciidoclet: An error has occurred in Javadoc report generation: 
[ERROR] Exit code: 4
[ERROR] javadoc: error - fatal error encountered: java.lang.NoClassDefFoundError: org/asciidoctor/Asciidoctor$Factory
[ERROR] javadoc: error - Please file a bug against the javadoc tool via the Java bug reporting page
[ERROR] (https://bugreport.java.com) after checking the Bug Database (https://bugs.java.com)
[ERROR] for duplicates. Include error messages and the following diagnostic in your report. Thank you.
[ERROR] java.lang.NoClassDefFoundError: org/asciidoctor/Asciidoctor$Factory
[ERROR]         at org.asciidoctor.asciidoclet.AsciidoctorConverter.<init>(AsciidoctorConverter.java:67)
[ERROR]         at org.asciidoctor.asciidoclet.Asciidoclet.run(Asciidoclet.java:214)
[ERROR]         at jdk.javadoc/jdk.javadoc.internal.tool.Start.parseAndExecute(Start.java:588)
[ERROR]         at jdk.javadoc/jdk.javadoc.internal.tool.Start.begin(Start.java:432)
[ERROR]         at jdk.javadoc/jdk.javadoc.internal.tool.Start.begin(Start.java:345)
[ERROR]         at jdk.javadoc/jdk.javadoc.internal.tool.Main.execute(Main.java:63)
[ERROR]         at jdk.javadoc/jdk.javadoc.internal.tool.Main.main(Main.java:52)
[ERROR] Caused by: java.lang.ClassNotFoundException: org.asciidoctor.Asciidoctor$Factory
[ERROR]         at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:476)
[ERROR]         at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:594)
[ERROR]         at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:527)
[ERROR]         ... 7 more
[ERROR] Command line was: /home/asalgadr/.sdkman/candidates/java/11.0.20-tem/bin/javadoc -J--add-exports=jdk.javadoc/jdk.javadoc.internal.tool=ALL-UNNAMED -J--add-exports=jdk.compiler/com.sun.tools.javac.parser=ALL-UNNAMED -J--add-exports=jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED -J--add-exports=jdk.compiler/com.sun.tools.javac.model=ALL-UNNAMED -Xdoclint:all,-html,-accessibility @options @packages
[ERROR] 
[ERROR] Refer to the generated Javadoc files in '/home/asalgadr/github/asciidoctor-maven-examples/asciidoclet/target/site/apidocs' dir.
```

--- a/pom.xml
+++ b/pom.xml
@@ -94,7 +94,6 @@
                         <mavenExecutorId>forked-path</mavenExecutorId>
                     </configuration>
                 </plugin>
-                <!-- default plugins for build, if not defined in parent-pom -->
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-resources-plugin</artifactId>
@@ -241,23 +240,6 @@
                     <useDefaultExcludes>true</useDefaultExcludes>
                     <strictCheck>true</strictCheck>
                 </configuration>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-shade-plugin</artifactId>
-                <version>3.5.2</version>
-                <configuration>
-                    <shadedArtifactAttached>true</shadedArtifactAttached>
-                    <shadedClassifierName>shaded</shadedClassifierName>
-                </configuration>
-                <executions>
-                    <execution>
-                        <phase>package</phase>
-                        <goals>
-                            <goal>shade</goal>
-                        </goals>
-                    </execution>
-                </executions>
             </plugin>
         </plugins>
     </build>

